### ISSUE TITLE
spiderwebs and scorpion

### ISSUE THREAD

**Issue by chriscauley** [chriscauley/hive.js#2]
Spider webs should apply to any piece that moves multiple tiles but must pass by the spider on the ground. Should not effect specials. Mosquitos are excluded from this list because they are restricted in the same way as whatever move they are copying. 

* No effect: Queen, beetle, ladybug, pill bug, mantis, fly, wasp, roach, centipede

* Ant: The ant can't move past the spider unless it is already touching the spider at the start of its turn (already done)

* Spider, scorpion: This one is tricky. Either spider/scorpion MUST move 3 spaces, in which case they could not move past a spider, or they would get trapped by spiderwebs, creating a situation where the spider/scorpion could move only one or two spaces.

* Dragonfly: Similar to spider and scorpion, does an dragonfly stepping next to a spider cause it to stop in the spider web or does it block the move entirely? How does this change if the dragonfly is holding a piece under itself as it moves?

# Scorpion sting

* No effect: queen, spider, ant, pill bug, scorpion, centipede.

* Currently blocks: Beetle, grasshopper, lady bug.

* Mantis: should not be able to pull scorpion under itself. (move should currently be blocked)

* Roach: should not be able to access tiles blocked by roach.

* Fly and wasp: Currently these both are magical teleporting pieces. They should step on the hive and be blocked by scorpion.

* Dragonfly: should be blocked by scorpion (cannot step over/land on)
